builtin/providers/aws/resource_aws_eip_test.go

```diff
@@ -12,6 +12,52 @@ import (
 	"github.com/hashicorp/terraform/terraform"
 )
 
+func TestAccAWSEIP_importEc2Classic(t *testing.T) {
+	resourceName := "aws_eip.bar"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSEIPDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSEIPInstanceConfig,
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateVerifyIgnore: []string{
+					"vpc",
```

Comment:
I'm curious why we're ignoring the import of `vpc`, I believe it's setting has an impact to how the resource behaves.